telemetry/definitions/commonDefinitions.json

```diff
@@ -550,6 +550,36 @@
             "description": "The type of the user trigger to send request to CodeWhisperer service",
             "allowedValues": ["OnDemand", "AutoTrigger"]
         },
+        {
+            "name": "codewhispererFirstCompletionLatency",
+            "type": "double",
+            "description": "The latency from user performs the triggering CW action to plugin call GenerateCompletions API for the first recommendation"
+        },
+        {
+            "name": "codewhispererEndToEndLatency",
+            "type": "double",
+            "description": "The latency from plugin start invocation to plugin call GenerateCompletions API to first completion shown in the IDE"
```

Comment:
The time it takes for the first completion to be shown in the IDE after the user performs the CW trigger action.